admin/controllers/school.js

```diff
@@ -54,7 +54,7 @@ const getPupils = async (req, res, next) => {
   const { pupils } = await fetchPupilsData(req.user.School)
   let pupilsFormatted = await Promise.all(pupils.map(async (p) => {
     const { foreName, lastName, _id } = p
-    const dob = moment(p.dob).format('DD/MM/YYYY')
+    const dob = moment(p.dob).format('DD MMM YYYY')
```

Comment:
There is a date service for almost this format.  Why use 'DD' rather than 'D'? The GDS standard is https://www.gov.uk/service-manual/design/dates which states no leading zero.

Suggest you use the DateService to prevent a proliferation of date formats.

Reply:
@jon-shipley updated